command/ca/certificate.go

```diff
@@ -139,6 +140,11 @@ $ step ca certificate foo.internal foo.crt foo.key \
 			acmeWebrootFlag,
 			acmeContactFlag,
 			acmeHTTPListenFlag,
+			cli.StringFlag{
+				Name: "k8ssa-token-path",
+				Usage: `Configure the <file> from which to read the kubernetes service account token.
+By default this location will be /var/run/secrets/kubernetes.io/serviceaccount/token.`,
+			},
```

Comment:
We should add the default value here using the `cli.StringFlag` property `Value`.

Reply:
This is also common for `step ca certificate/token/sign`, unify them in a variable.